src/compiler/abstract_compiler.nit

```diff
@@ -1594,39 +1594,39 @@ abstract class AbstractCompilerVisitor
 	fun int8_instance(value: Int8): RuntimeVariable
 	do
 		var t = mmodule.int8_type
-		var res = new RuntimeVariable("((int8_t){value.to_s})", t, t)
+		var res = new RuntimeVariable("INT8_C({value.to_s})", t, t)
```

Comment:
Why the change to a macro here? Are fixed_ints changed to their `fast` variant too?